docs/content/config/environment.md

```diff
@@ -286,6 +286,13 @@ Recipient address for logwatch reports if they are enabled.
 - **not set** => Use REPORT_RECIPIENT or POSTMASTER_ADDRESS
 - => Specify the recipient address(es)
 
+##### LOGWATCH_SENDER
+
+From address for logwatch reports if they are enabled.
```

Comment:
I'd rephrase this:

This variable sets the sender address (`FROM`) for logwatch reports if logwatch reports are enabled.

Reply:
I also just copied these lines from `PFLOGSUMM_SENDER`. 😏

How about a bit shorter form?
**Sender address (`FROM`) for logwatch reports if logwatch reports are enabled.**

Reply:
Fine with me :)

Reply:
Done.